### ISSUE TITLE
[Bug] Property 'renderToSVGString' does not exist on type 'EChartsType'.

### ISSUE THREAD

**Issue by han-ysys** [apache/echarts#17615]
### Version

5.3.3

### Steps to Reproduce

我想把chart导出成pdf，查到5.3.0之后有renderToSVGString这个方法，尝试了一下
```javascript
const echartsdiv = document.getElementById('chart-for-download') as HTMLElement
chartins = echarts.init(echartsdiv, undefined, {renderer: 'svg'})
console.log('renderToSVGString', chartins.renderToSVGString())
```

### Current Behavior

vscode报错

> Property 'renderToSVGString' does not exist on type 'EChartsType'.

然而在echarts.d.ts里是能找到renderToSVGString的定义的

> renderToSVGString(opts?: {
        useViewBox?: boolean;
    }): string;

### Expected Behavior

希望能够得到解决

**Comment by Ovilia** [comment]
I tested it with [test/ssr-animaiton-wave.htm](https://github.com/apache/echarts/blob/master/test/ssr-animaiton-wave.html) and it seems to be working. Please provide an online demo.

**Comment by han-ysys** [comment]
The code itself works. It does give SVG strings. But IDE keeps whining about type errors.

After several tries with different IDE and typescript versions(4.1.6 and 4.7.3), somehow all reach an agreement. If you are interested, what I did is open the project with Webstorm, find no complaints about 'renderToSVGString', and then go back to VSCode. I noticed VScode was using TS 4.7.3 while Webstorm was using 4.1.6. So I switched it to 4.1.6. Everything goes fine now. The thing is if I switch it back to TS 4.7.3, it won't report any error.

It seemed to be just something wrong with my environment. Sorry to bother you all. Thank you!
